backend/notenverwaltung/src/main/java/ch/lalumamesh/notenverwaltung/validator/GroupValidator.java

```diff
@@ -0,0 +1,20 @@
+package ch.lalumamesh.notenverwaltung.validator;
+
+import ch.lalumamesh.notenverwaltung.api.IValidator;
+import ch.lalumamesh.notenverwaltung.model.GraduationGroup;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+
+@Component
+public class GroupValidator implements IValidator<GraduationGroup> {
+    @Override
+    public void validate(GraduationGroup object, Errors errors) {
+        if (object.getParentGrad() == null && object.getParentGroup() == null) {
```

Comment:
Naming (getParentGrade() ?)

now one must be null. intended?

Reply:
typo

Reply:
intended, a group can not have two parents, either a group or a graduation, but this is no longer relevant, considering that groups no longer exist.